main/webservices/soap_session.php

```diff
@@ -81,6 +81,32 @@
 );
 
 $s->register(
+	'WSSession.SubscribeTeacherToSession',
```

Comment:
The web service should be called "SubscribeTeacherToSessionCourse", otherwise it feels like you are subscribing him to the session, not the *course* inside that session